com.unity.visualeffectgraph/CHANGELOG.md

```diff
@@ -76,6 +76,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Inspector group headers now have a better indentation and alignment
 - Zoom and warning icons were blurry in the "Play Controls" and "Visual Effect Model" scene overlays
 - Random crash using subgraph [Case 1345426](https://issuetracker.unity3d.com/product/unity/issues/guid/1345426/)
+- Graph Editor now creates the window's UI on the correct callback
```

Comment:
No need for a changelog in that case. It does not bring anything to users and there's no bug assicoated with it